common/app/views/fragments/head.scala.html

```diff
@@ -14,6 +14,22 @@
     <script src="@Static("javascripts/components/html5shiv/html5shiv.js")"></script>
 <![endif]-->
 
+@* provide some ipad users with the core view of fronts *@
+<script>
```

Comment:
That would not be a good test then, one major question to ask is "Do people who see the core version come back more often?"